frontend/containers/breadcrumbs/component.tsx

```diff
@@ -0,0 +1,128 @@
+import { FC } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { Paths } from 'enums';
+
+import type { BreadcrumbsProps } from './types';
+
+export const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  className,
+  substitutions: propSubstitutions = {},
+}: BreadcrumbsProps) => {
+  const intl = useIntl();
+  const { route, asPath } = useRouter();
+
+  // Predefined substitutions for this project. Ideally this would live in a constants file,
+  // but due to the need to use `react-intl` they're kept here.
+  const substitutions = {
+    'project-developer': {
+      name: intl.formatMessage({ defaultMessage: 'Project Developers', id: '+K9fF0' }),
+      link: Paths.ProjectDevelopers,
+    },
+    project: {
+      name: intl.formatMessage({ defaultMessage: 'Projects', id: 'UxTJRa' }),
+      link: Paths.Projects,
+    },
+    'open-call': {
+      name: intl.formatMessage({ defaultMessage: 'Open Calls', id: 'wpyHb9' }),
+      link: Paths.OpenCalls,
+    },
+    investor: {
+      name: intl.formatMessage({ defaultMessage: 'Investors', id: 'zdIaHp' }),
+      link: Paths.Investors,
+    },
+  };
+
+  const routeSegments: string[] = route.split('/').slice(1);
+  const pathSegments: string[] = asPath.split('/').slice(1);
+
+  const nameFromPathname = (text: string): string =>
+    text
+      .replace(/(-|_)/, ' ') // Change Dashes and Underscores into spaces
+      .split(' ') // Split into an array of words
+      .map((word) => word.charAt(0).toUpperCase() + word.slice(1)) // Uppercase words' first letters
```

Comment:
Be careful with the `chartAt` function. I believe it won't work as you expect for certain characters (such as emojis) which are actually composed of several characters. In this case however, I don't think any changes are needed.